src/shared/sol-conffile.c

```diff
@@ -37,9 +37,16 @@
 #include "sol-memmap-storage.h"
 #endif
 
+struct sol_alias_ctx {
+    char *node;
```

Comment:
ah, and the data you can abuse and add to the arena in this file, so no need to free manually